### ISSUE TITLE
doc/howto/using-twistedweb.xhtml is a bit incorrect about indices

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11398]
|<img alt="dotz's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| dotz reported|
|-|-|
|Trac ID|trac#378|
|Type|defect|
|Created|2003-10-30 20:15:56Z|

<details><summary>Searchable metadata</summary>

```
trac-id__378 378
type__defect defect
reporter__dotz dotz
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1067544956000000 1067544956000000
changetime__1469142441986746 1469142441986746
version__ 
owner__ 
cc__dotz
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="dotz's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|dotz commented|
|-|-|

```
#!html
<pre>
There are no 'indices'. There are 'indexNames'. Please apply the patch.

diff -u -r1.6 using-twistedweb.xhtml
--- using-twistedweb.xhtml      13 Oct 2003 08:15:27 -0000      1.6
+++ using-twistedweb.xhtml      30 Oct 2003 14:14:01 -0000
@@ -236,7 +236,7 @@
 
 &lt;p>&lt;code>File&lt;/code> resources, be they root object or children thereof,
 have two important attributes that often need to be modified:
-&lt;code>indices&lt;/code> and &lt;code>processors&lt;/code>. &lt;code>indices&lt;/code>
+&lt;code>indexNames&lt;/code> and &lt;code>processors&lt;/code>. &lt;code>indexNames&lt;/code>
 determines which files are treated as &lt;q>index files&lt;/q> -- served
 up when a directory is rendered. &lt;code>processors&lt;/code> determine how
 certain file extensions are treated.&lt;/p>
@@ -250,7 +250,8 @@
 from twisted.web import static, server, script
 
 root = static.File("/var/www/htdocs")
-root.indices=['index.rpy']
+root.indexNames=['index.rpy']
+# or: root.indexNames.append('index.php')
 root.processors = {'.rpy': script.ResourceScript}
 application = service.Application('web')
 sc = service.IServiceCollection(application)
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)|@moshez commented|
|-|-|

```
#!html
<pre>
/cvs/Twisted/doc/howto/using-twistedweb.xhtml,v  &lt;--  using-twistedweb.xhtml
new revision: 1.7; previous revision: 1.6

Fixed
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset 71f7b90c4905d984fc46aca7f455fbd6113607bd
```
#!CommitTicketReference repository="" revision="71f7b90c4905d984fc46aca7f455fbd6113607bd"
Merge pull request #378 from twisted/8644-rodrigc-write_sequence-test

Author: rodrigc
Reviewer: scop
Fixes: #8644

Add test for twisted.internet.udp.Port.writeSequence
```
